wrapper/symbols/crypto_pwhash_scryptsalsa208sha256_str.json

```diff
@@ -24,5 +24,5 @@
         ],
         "target": "libsodium._crypto_pwhash_scryptsalsa208sha256_str(hashed_password_address, password_address, password_length, 0, opsLimit, 0, memLimit) | 0",
         "assert_retval": [{"condition": "=== 0", "or_else_throw": "invalid usage"}],
-        "return": "libsodium.Pointer_stringify(hashed_password_address)"
+        "return": "libsodium.UTF8ToString(hashed_password_address, password_length)"
```

Comment:
I don't think `password_length` is the length of the hash.

Reply:
You're probably right. Should it be `libsodium._crypto_pwhash_scryptsalsa208sha256_strbytes()`?

Reply:
These strings are `\0`-terminated, so maybe we don't need the length at all :)

Reply:
Just read this on [emscripten](https://emscripten.org/docs/api_reference/preamble.js.html):
> N.B. mixing frequent uses of UTF8ToString() with and without maxBytesToRead may throw JS JIT optimizations off, so it is worth to consider consistently using one style or the other.

Not sure if it matters, but removed them from all the calls :)